g2p_registry_base/models/reg_relationship.py

```diff
@@ -27,6 +32,9 @@ class G2PRegistrantRelationship(models.Model):
     disabled_by = fields.Many2one("res.users")
     start_date = fields.Datetime()
     end_date = fields.Datetime()
+    registrant1_as_str = fields.Char(related="registrant1.name")
+    registrant2_as_str = fields.Char(related="registrant2.name")
```

Comment:
Why do you need this? We do not need their name, just the IDs.

Reply:
Yeah I removed this.